htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/Table.java

```diff
@@ -49,68 +45,70 @@ public Table(WebElement wrappedElement) {
      * @return List with text values of table heading elements.
      */
     public List<String> getHeadingsAsString() {
-        return convert(getHeadings(), toTextValues());
+        return getHeadings().stream()
+                .map(WebElement::getText)
+                .collect(toList());
     }
 
     /**
-     * Returns table cell elements grouped by rows.
+     * Returns table cell elements ({@code <td>}), grouped by rows.
      *
      * @return List where each item is a table row.
      */
     public List<List<WebElement>> getRows() {
-        List<List<WebElement>> rows = new ArrayList<>();
-        List<WebElement> rowElements = getWrappedElement().findElements(By.xpath(".//tr"));
-        for (WebElement rowElement : rowElements) {
-            rows.add(rowElement.findElements(By.xpath(".//td")));
-        }
-        return rows;
+        return getWrappedElement()
+                .findElements(By.cssSelector("tr"))
```

Comment:
changing selector type may produce unwanted side effects for some legacy browsers (which used a lot for some enterprise web apps). So please move change from xpath to css to separate PR